front/pages/w/[wId]/assistant/gallery.tsx

```diff
@@ -143,31 +143,31 @@ export default function AssistantsGallery({
   const tabs: {
```

Comment:
Since `current` is conditional based on the `agentsGetView` state, this should be behind a `useMemo`.